mb_add_remix_credit_links.user.js

```diff
@@ -60,6 +60,9 @@ function addStyleElement() {
 
 function addRemixCreditLinks() {
     const recordings = document.getElementsByClassName('recording');
+    const releaseArtists = Array.from(
+      document.getElementsByClassName('subheader')[0].getElementsByTagName('bdi')
+    ).slice(0, -1).map(bdi => bdi.innerText); // slice "see all versions of this release"
 
```

Comment:
Nice example of the use of `map()`. I regularly use functional style programming in ruby but my javascript knowledge is about 10 years out of date so this this is helpful. Thanks! :+1: